linkerd/namer/fs/src/main/scala/io/l5d/fs.scala

```diff
@@ -2,26 +2,21 @@ package io.l5d
 
 import com.fasterxml.jackson.annotation.JsonIgnore
 import com.twitter.finagle.{Stack, Path}
-import io.buoyant.linkerd.config.Parser
 import io.buoyant.linkerd.config.types.Directory
 import io.buoyant.linkerd.namer.fs.WatchingNamer
 import io.buoyant.linkerd.{NamerConfig, NamerInitializer}
-import io.l5d.fs.FsConfig
 
-class fs extends NamerInitializer {
-  val configClass = Parser.jClass[FsConfig]
-  val configId = "io.l5d.fs"
+class fsInitializer extends NamerInitializer {
```

Comment:
and in fact MarathonInitializer is capitalized... so let's be consistent.  and i'd prefer that we go camelcase with K8sInitializer, FsInitializer, etc.